go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/osbuild/osbuild-composer
 
-go 1.21
+go 1.21.0
+
+toolchain go1.22.5
```

Comment:
I'm not sure if we can do this, won't mock builds fail if 1.22 isn't available?

Reply:
Ah frik. Sorry that was a mistake on my part. I think I accidentally updated go on my system

Reply:
Should be fixed now